src/pudl/etl.py

```diff
@@ -371,8 +396,22 @@ def _etl_glue(glue_settings: GlueSettings) -> dict[str, pd.DataFrame]:
 
     # Add the EPA to EIA crosswalk, but only if the eia data is being processed.
     # Otherwise the foreign key references will have nothing to point at:
+    ds = Datastore(**ds_kwargs)
```

Comment:
I think the reason it's outside the conditional is in case we added other glue tables that would also require the ds argument. We don't have any now though. Where do you think it should live?

Reply:
ah okay that makes sense. its a lil weird but is more open to expansion and not at all time/memory intensive.